building/devices/zynqmp.rst

```diff
@@ -0,0 +1,92 @@
+
+#######################
+QUEMU for ZynqMP zcu102
+#######################
+The instructions here will tell how to run OP-TEE using QEMU for ZynqMP zcu102 board. 
+
+Setting up the toolchain
+************************
+
+This build chain heavily relies on Petalinux 2018.2 therefore the fisrt step is
+to download and install the Petalinux 2018.2 toolchain from the Xilinx website.
+By default, the makefile considers that Petalinux is installed in ``/opt/Xilinx/petalinux_2018_2``.
+Then, you have to download the zcu102 BSP file from the Xilinx website.
+
+You may have to create a free Xilinx account to proceed with the two previous tests.
+
+Configuring and building
+************************
+
+Fisrt, create a new directory which will be used as root directory:
+
+.. code-block:: none
+
+	# mkdir -p ~/petalinux-optee
+	# cd ~/petalinux-optee
+
+Then, copy the zcu102 BSP file into the newly created directory:
+
+.. code-block:: none
+
+	# cp ~/Downloads/xilinx-zcu102-v2018.2-final.bsp .
+
+Git clone the ``build`` repository of the OPTEE project and source the Petalinux settings:
+
+.. code-block:: none
+
+	# git clone https://github.com/OP-TEE/build
+	# cd ./build
+	# source /opt/Xilinx/petalinux_2018_2/settings.sh
+
+Finally, type the following commands to create, patch, configure and build the Petalinux project.
+Petalinux is a powerfull but very slow tool, each command may take a while according to the capabilities of your computer.
+
+.. code-block:: none
+	
+	# make -f zynqmp.mk petalinux-create
+	# make -f zynqmp.mk petalinux-config
+	# make -f zynqmp.mk petalinux-build
+
+Once the last command ends up you are ready to run the QEMU tool. Type the following command:
+
+.. code-block:: none
+
+	# make -f zynqmp.mk qemu
+
+The QEMU will start and launch Petalinux distribution. At the end of the boot process,
+log in using the username ``root`` and the password ``root``.
+Start the OPTEE Normal World service and run xtest :
+
+.. code-block:: none
+	
+	# tee-supplicant &
+	# xtest
+	
+You can close the QEMU session at any time by typing ``Ctrl-A+C`` and entering the quit ``command``.
+
+QUEMU for others ZynqMP boards
+******************************
+
+This makefile supports others ZynqMP boards:
+
+	* zcu104
+	* zcu106
+
+To use this makefile with these boards, you have to download the corresponding BSP and
+add the option ``PLATFORM=zcu104`` or ``PLATFORM=zcu106`` to each make command.
+
+.. code-block:: none
+	 
+	# make -f zynqmp.mk PLATFORM=zcu=106 petalinux-create
+	# make -f zynqmp.mk PLATFORM=zcu=106 petalinux-config
+	# make -f zynqmp.mk PLATFORM=zcu=106 petalinux-build
+	# make -f zynqmp.mk PLATFORM=zcu=106 qemu
+
+Customizing the Peatlinux distribution
+**************************************
+
+You can customize the Petalinux project (i.e. kernel, rootfs, ...) as any standard Petalinux project.
+Just enter the project directory and type your commands. For additional information, refer to Petalinux Tool Documentation (UG1114) from Xilinx.
```

Comment:
Do you have a link to this documentation? If you have please put that a the bottom at the document and refer to it in the text here (see other rst-files in optee_docs to figure out how it is done in other places).